tests/packagedcode/test_recognize.py

```diff
@@ -196,3 +196,9 @@ def test_recognize_go_sum(self):
         packages = recognize_package_data(test_file)
         assert packages
         assert isinstance(packages[0], models.PackageData)
+
+    def test_recognize_rpmdb_sqlite(self):
+        test_file = self.get_test_loc('rpm/rpmdb.sqlite')
```

Comment:
I wonder if we could get a much smaller test file for this? we only need a few packages not everything.